frontend/src/hooks/useBoard.tsx

```diff
@@ -114,11 +115,37 @@ const useBoard = ({ autoFetchBoard = false }: AutoFetchProps): UseBoardType => {
     },
   });
 
+  const updateBoardPhase = (data: UpdateBoardPhase) => {
+    const getBoardQuery = (id: string | undefined) => ['board', { id }];
```

Comment:
```suggestion
setReady(false)
    const getBoardQuery = (id: string | undefined) => ['board', { id }];
```

We need this "ready" state to make the socket queue of operations ready to handle the next task. Or on the other hand to lock the queue. 
To make this works you need to add the following line on the top of the file near the other `useState`. 
`const setReady = useSetRecoilState(operationsQueueAtom);`